app/promotion_keys.py

```diff
@@ -16,6 +15,7 @@ class PromotionKeysHandler(BaseHandler):
     @gen.coroutine
     def post(self):
         showtime_id = self.get_argument('showtime_id', None)
```

Comment:
If we are going to have `showtime_id` be the convention, could you change [app/showtimes.py:26](https://github.com/QuantifiedSelfless/quantifiedSelf/blob/improvement/bulk-promo/app/showtimes.py#L26) to also use this?